pkg/release-controller/release.go

```diff
@@ -67,12 +67,24 @@ func (c imageInfoConfig) GenerateDigestPullSpec() string {
 	}
 	return fmt.Sprintf("%s@%s", strings.Split(c.Name, ":")[0], c.Digest)
 }
+func (r *Release) HasInconsistencies() bool {
+	for _, tag := range r.Source.Spec.Tags {
+		if _, ok := tag.Annotations[ReleaseAnnotationInconsistency]; ok {
+			return true
+		}
+	}
+	if _, ok := r.Source.ObjectMeta.Annotations[ReleaseAnnotationInconsistency]; ok {
+		return true
+	}
```

Comment:
Does ART actually use the Annotation on the imagestream itself?  I only saw it on the `.Spec.Tags`...

Reply:
I believe so, but I will double-check.

It should be here (inconsistency tag for payload-wide issues in an annotation on the imagestream itself): https://github.com/openshift/doozer/blob/6e29df43a6fa53efa4eddc037e3630e876f16a53/doozerlib/cli/release_gen_payload.py#L881